src/mygrad/io.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+import typing
+import mygrad as mg
+from mygrad.tensor_base import tensor, Tensor
+
+
+def save(filename : str, tensor : Tensor) -> None:
```

Comment:
Let's change `filename` to `file` so that we have "parity" with NumPy's `save` function